src/com/returnhightnumbersolution/ReturnHightNumberSolution.java

```diff
@@ -0,0 +1,30 @@
+package com.returnhightnumbersolution;
+
+public class ReturnHightNumberSolution {
```

Comment:
I told you to use the existing class you created `algoSolution`, just create a function in it not a new class